_rules/html-page-lang-matches-default-ucwvc8.md

```diff
@@ -0,0 +1,280 @@
+---
+id: ucwvc8
+name: HTML page language subtag matches default language
+rule_type: atomic
+description: |
+  This rule checks that the primary language subtag of the page language matches the default language of the page
```

Comment:
Based on https://github.com/act-rules/act-rules.github.io/pull/1334/files#r464284953

Couldn't we omit "subtag" here as well?


```suggestion
  This rule checks that the primary language of the page language matches the default language of the page
```

Reply:
See my comment below. You do have to say subtag here. The "primary language of the page" in itself is not the same as the "primary language of the language tag"

Reply:
Fair enough.